Source/Core/Core/NetPlayClient.cpp

```diff
@@ -514,7 +533,7 @@ void NetPlayClient::ThreadFunc()
 				break;
 			}
 		}
-
+		Common::SleepCurrentThread(1);
```

Comment:
Through this sleep make it faster right now, it won't fix the problem and is completely orthogonal to this PR. A possible fix is done in PR #2198 
I'd suggest to just not touch this line here.